sklearn/ensemble/weight_boosting.py

```diff
@@ -903,6 +921,21 @@ class AdaBoostRegressor(BaseWeightBoosting, RegressorMixin):
     feature_importances_ : array of shape = [n_features]
         The feature importances if supported by the ``base_estimator``.
 
+    Examples
+    --------
+    >>> from sklearn.ensemble import AdaBoostRegressor
+    >>> from sklearn.datasets import make_regression
+    >>> X, y = make_regression(n_features=4, n_informative=2,
+    ...                        random_state=0, shuffle=False)
+    >>> regr = AdaBoostRegressor(random_state=0, n_estimators=100)
+    >>> regr.fit(X, y)  # doctest: +NORMALIZE_WHITESPACE
+    AdaBoostRegressor(base_estimator=None, learning_rate=1.0, loss='linear',
+            n_estimators=100, random_state=0)
+    >>> print(regr.feature_importances_)
```

Comment:
please use ` # doctest: +ELLIPSIS` here and keep all the following floating points to max 4 digits. This level of precision causes us headaches when we try to build for all the platforms we do. (i.e. `0.27885832` -> `0.2788...`)

Reply:
That might exists in a bunch of places